### PR TITLE
Proper workflow for WSL users

### PR DESCRIPTION
**Pull request by augustogoulart**
Fix for #425

### CODE REVIEW

**Comment by augustogoulart** [comment]
@taleinat

**Comment by taleinat** [comment]
@zware

> I can't reproduce the issue, and the advice to fix it is not clear to me anyway. Can you better describe the steps you took, what happened, and what you expected instead?

FWIW I think the discussion about reproduction should be done on the issue rather than here.

File: setup.rst [hunk 1]
```diff
@@ -236,7 +236,10 @@ find git for Windows on the **Individual components** tab of the installer.
   may not be configured on recent versions of Windows, such as Windows 10). If
   you are building on a recent Windows version, use the Control Panel (Programs
   | Programs and Features | Turn Windows Features on or off) and ensure that the
-  entry ".NET Framework 3.5 (includes .NET 2.0 and 3.0)" is enabled.
+  entry ".NET Framework 3.5 (includes .NET 2.0 and 3.0)" is enabled. If you are
```
Comment (taleinat) [inline setup.rst]: This "Note" doesn't seem like an appropriate place for this info. I'd perhaps place it at the end of this section, before/after the note about Python 2.7 using a different version of Visual Studio.

**Comment by augustogoulart** [comment]
Change requests and suggestion included.

File: setup.rst [hunk 2]
```diff
@@ -256,6 +256,12 @@ to build.
    `PCbuild directory`_ you will also need Visual Studio 2010, see the `2.7
    readme`_ for more details.  If you have VS 2008 but not 2010 you can use the
    build files in the `PC/VS9.0 directory`_, see the `VS9 readme`_ for details.
+   If you are using the Windows Subsystem for Linux (WSL), clone the repository
+   from a native Windows terminal program like cmd.exe command prompt or
+   PowerShell and use a build of git targeted for Windows, e.g. the official one
```
Comment (willingc) [inline setup.rst]: ~and~ as well as

Comment (augustogoulart) [reply setup.rst]: Thanks, @willingc! it's fixed.

**Comment by augustogoulart** [comment]
@taleinat done!

**Comment by willingc** [comment]
@zware Any objection to merging?

**Comment by augustogoulart** [comment]
Thanks, @taleinat. Sentence splitted!

**Comment by matrixise** [comment]
@zware could you review this PR? I think we could merge it.

**Comment by willingc** [comment]
@matrixise See Zach's response on issue: https://github.com/python/devguide/issues/425#issuecomment-431875473

**Comment by matrixise** [comment]
oh sorry @willingc I was wrong, thank you for your ping.

**Comment by willingc** [comment]
I'm going to go ahead and merge this PR since this note appears useful to others. 

I'll leave the original issue open, @zware, for any further comments which you may have.

Thanks @augustogoulart for the PR.

**Pull request merged**